pytensor/link/c/cmodule.py

```diff
@@ -2716,11 +2717,77 @@ def default_blas_ldflags():
 
     """
     warn_record = []
+
+    def check_required_file(paths, required_regexs):
+        found = []
+        for req in required_regexs:
+            for path in paths:
+                m = re.search(req, path.name)
+                if m:
+                    found.append(m.string[slice(*m.span())])
+                    break
+            raise RuntimeError(f"Required file {req} not found")
+        return found
+
+    def get_default_blas_libraries(library_dirs):
+        for library_dir in library_dirs:
+            all_libs = [
+                l
+                for l in pathlib.Path(library_dir).iterdir()
+                if l.suffix in (".a", ".so", ".dll", ".lib")
+            ]
+            try:
+                # First try to find mkl
+                found_libs = check_required_file(
+                    all_libs,
+                    required_regexs=["mkl_core", "mkl_rt", "mkl_(gnu|intel)_thread"],
+                )
+                return found_libs
+            except RuntimeError:
+                pass
+            try:
+                # Second try to find openblas
```

Comment:
What about accelerate?

Reply:
That wasn't ever used as a default. I don't know how to test it. I'm not even sure how to test whether LAPACK works.

Reply:
Do we even link against LAPACK in the c-backend? I couldn't find much reference. https://github.com/search?q=repo%3Apymc-devs%2Fpytensor%20lapack&type=code

Reply:
Just talked to @aseyboldt, who said that yes we do use LAPACK for all the LA stuff, and usually LAPACK would come shipped with blas.